girder/utility/plugin_utilities.py

```diff
@@ -87,6 +87,30 @@ def loadPlugins(plugins, root, appconf, apiRoot=None, curConfig=None):
 
     return root, appconf, apiRoot
 
+def getPluginParentDir(name, curConfig=None):
+    """
+    Finds the directory a plugin lives in and returns it. This throws
+    an exception if it can't find a directory named name in any of the
+    set plugin_directory paths.
+
+    Also throws an exception if it finds multiple parents, i.e. the plugin
+    named name lives in more than one place.
+
+    :params name: The name of the plugin (i.e. its directory name)
+    :type name: str
+    """
+    parentDirs = set()
+
+    for potentialParentDir in getPluginDir(curConfig):
+        if os.path.exists(os.path.join(potentialParentDir, name)):
+            parentDirs.add(potentialParentDir)
+
+    if len(parentDirs) > 1:
+        raise Exception('Plugin directory %s exists in multiple paths' % name)
```

Comment:
This shouldn't be an exception, we should just use the one that is found first on the path, which is probably what most people would expect if they are familiar with path searching behavior.